sokol_app.h

```diff
@@ -1086,9 +1108,15 @@ int main(int argc, char* argv[]) {
 #endif /* SOKOL_NO_ENTRY */
 
 _SOKOL_PRIVATE void _sapp_macos_update_dimensions(void) {
-    const CGSize fb_size = [_sapp_view_obj drawableSize];
-    _sapp.framebuffer_width = fb_size.width;
-    _sapp.framebuffer_height = fb_size.height;
+    #if defined(SOKOL_METAL)
+        const CGSize fb_size = [_sapp_view_obj drawableSize];
+        _sapp.framebuffer_width = fb_size.width;
+        _sapp.framebuffer_height = fb_size.height;
+    #elif defined(SOKOL_GLCORE33)
+        const NSRect fb_rect = [_sapp_view_obj convertRectToBacking:[_sapp_view_obj frame]];
+        _sapp.framebuffer_width = fb_rect.size.width;
+        _sapp.framebuffer_height = fb_rect.size.height;
```

Comment:
According to [this document](https://developer.apple.com/library/archive/documentation/GraphicsAnimation/Conceptual/HighResolutionOSX/CapturingScreenContents/CapturingScreenContents.html), `[_sapp_view_obj bounds]` denotes the number of pixels in the framebuffer. Should frame and bounds be switched for this logic? I don't have a Mac with a retina display so I can't test this.

Reply:
I'll do some testing and report back, it may be that the GL backend didn't have correct handling for HighDPI, and that some code confuses window size and framebuffer size.

Reply:
Hmm, both the ```frame``` and ```bounds``` method return the same (wrong) values, I remember that this is another thing that broke under Mojave, somehow the framebuffer is always initialized at Retina resolution even when this is not requested.

I'd say: keep it as it is for now, that's something I'll need to care about when I'm looking into the vsync problem.

Reply:
Ok, let's merge it :)